ecms_base/themes/custom/ecms/ecms.theme

```diff
@@ -479,3 +479,62 @@ function ecms_preprocess_block(array &$variables): void {
     }
   }
 }
+
+/**
+ * Implements hook_preprocess_paragraph().
+ */
+function ecms_preprocess_paragraph__numbered_step_item(&$variables): void {
+
```

Comment:
Should there be defaults set here for `$variable['list_start']` and `$variable['list_end']`? If they are initialized here to false we can be sure they are always passed to the twig template